tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -672,9 +672,7 @@ def test_correct_compressor_inferred(
     if is_24:
         weights = _make_24_sparse(weights)
     else:
-        weights[0, :] = torch.ones(
-            4,
-        )  # guarantee not 24 sparse
+        weights[0, :] = torch.ones((4,))  # guarantee not 24 sparse
```

Comment:
Does this need a rebase? 🤔

Reply:
ugh, sorry, dammit i keep using black/isort instead of flake8. they format this line differently